pkgs/applications/networking/cluster/nixops/default.nix

```diff
@@ -43,14 +43,26 @@ let
       inherit withPlugins python;
     };
   }));
-in withPlugins (ps: [
-  ps.nixops-aws
-  ps.nixops-digitalocean
-  ps.nixops-encrypted-links
-  ps.nixops-gce
-  ps.nixops-hercules-ci
-  ps.nixops-hetzner
-  ps.nixops-hetznercloud
-  ps.nixops-libvirtd
-  ps.nixops-vbox
-])
+in lib.mapAttrs
+  (name: plugnames: withPlugins (ps: map (k: builtins.getAttr k ps) plugnames) )
+  {
+    minimal      = [];
+    aws          = [ "nixops-aws" ];
+    digitalocean = [ "nixops-digitalocean" ];
+    gce          = [ "nixops-gce" ];
+    hercules-ci  = [ "nixops-hercules-ci" ];
```

Comment:
This one is commercially supported and is typically used alongside other plugins.

Reply:
Note that it's still fully available in `nixops_unstable`.